### PR TITLE
updated caching (untested)

### PR DESCRIPTION
**Pull request by Tieske**

### CODE REVIEW

File: kong/plugins/correlation-id/handler.lua [hunk 1]
```diff
@@ -11,7 +11,8 @@ local worker_uuid
 local worker_counter
 
 local fmt = string.format
-local ngx = ngx
+local now = ngx.now
+local worker_pid = tostring(ngx.worker.pid())
```
Comment (thibaultCha) [inline kong/plugins/correlation-id/handler.lua]: You could just use `"%d"` in `string.format` instead.

Comment (Tieske) [inline kong/plugins/correlation-id/handler.lua]: this runs once, string.format runs on each request

Comment (thibaultCha) [inline kong/plugins/correlation-id/handler.lua]: That it runs with %d or %s is negligible, and more concise.

**Comment by WALL-E** [comment]
手动应用

**Pull request closed without merging**